auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -68,16 +68,19 @@
         data-test-tag="add-business"
       >
         <template v-slot:text>
-          <p>Add an existing business by entering the Incorporation Number <span class="wb">and associated Passcode.</span></p>
+          <p>
+            Add an existing business by entering the Incorporation Number <span class="wb">and associated
```

Comment:
Apparently the "wb" class is to control word breaking. I don't think it's declared anywhere but I didn't want to make too make changes.